<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrayscaleForm</class>
 <widget class="QWidget" name="GrayscaleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grayscale</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="1" colspan="3">
    <widget class="QPushButton" name="buttonSave">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Save gray image as ...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QPushButton" name="buttonOpenAndGray">
     <property name="text">
      <string>Open image and apply grayscale</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelOriginalImage">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </item>
       <item>
        <widget class="QLabel" name="labelGrayImage">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
